tests/test_getPackageDir.py

```diff
@@ -19,17 +19,15 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
If we're wrapping the C++ solely so it can be tested in Python, that seems a bit weird.  We should write a C++ test for `getPackageDir()`.

Reply:
Or I can call it `_getPackageDir` and only use it in the test.

Reply:
But the point of the function is to be accessible from C++.  No one is ever supposed to use a pybind11-wrapped version of it; they should be using `lsst.utils.getPackageDir()` instead.  Possible C++ test sent under separate cover.

Reply:
I've added your test code.